src/view/com/post-thread/PostThreadItem.tsx

```diff
@@ -203,6 +209,13 @@ let PostThreadItemLoaded = ({
       ),
     [post, langPrefs.primaryLanguage],
   )
+  const postLang = useMemo(
+    () =>
+      sanitizedLanguage && !isPostInLanguage(post, [sanitizedLanguage])
+        ? getPostLanguage(post)
+        : undefined,
+    [post, sanitizedLanguage],
+  )
```

Comment:
Can you explain why you're doing this instead of just putting the post's declared language in the field? If there are multiple languages declared and only one is supported, we can either abort or just take the first

Reply:
@pfrazee Sorry, I don't know much about Typescript, and I wrote this by referencing code from elsewhere, so it's going to be hard to explain what I wrote clearly...

If you prefer to implement this feature in a different way, please feel free to close this PR. Thank you!